starter-dev-backend/types/graphql.ts

```diff
@@ -59,25 +79,17 @@ export type Repo = {
   id?: Maybe<Scalars['ID']>;
   language?: Maybe<Scalars['String']>;
   name?: Maybe<Scalars['String']>;
-  owner?: Maybe<Owner>;
+  owner?: Maybe<User>;
   private?: Maybe<Scalars['Boolean']>;
   stargazers_count?: Maybe<Scalars['Int']>;
   title?: Maybe<Scalars['String']>;
   updated_at?: Maybe<Scalars['String']>;
 };
 
-/** A User object */
+/** A Owner object used in Repo */
```

Comment:
Suggestion: Grammar 😛

```suggestion
/** An Owner object used in Repo */
```